src/candidateInfo.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './candidateInfo.css';
+import CandidateAnalysis from './candidateAnalysis';
+import ModalSplash from './modalSplash.js';
+
+function CandidateInfo(props) {
+
+    const handleClick = (e) => {
+        e.preventDefault();
+        props.onClickModalNav(e);
+    }
+
+    return ( 
+        <div className="modal-candidate-info">
+            <div className="modal-nav-container">
+                <div className="modal-nav">
+                    <div id="1" data-name="biden" onClick={handleClick} className={"modal-nav-item " + (props.candidateName === "biden" ? 'nav-selected' : '')}>Biden {props.row.biden.score}</div>
```

Comment:
Doesn't need to be addressed now, but I wonder if we should pull this in from the Data module in case the team wants to add more candidates

Reply:
Yep, definitely want to start clearing this out.. I'm thinking with the next major 'version' of the SC. We can introduce a value in the State that represents the 3 currently selected candidates, and update impacted components from there.